gnat2goto/driver/goto_utils.adb

```diff
@@ -136,14 +135,14 @@ package body GOTO_Utils is
    end Int64_T;
 
    Char_T : Irep := Ireps.Empty;
-   function Uint8_T return Irep
+   function Int8_T return Irep
```

Comment:
This `uint8_t` doesn't have a replacement because this was the only use for it (as per correspondence with @xbauch )